state_management/go/sdk/order-processor/app.go

```diff
@@ -18,7 +18,7 @@ func main() {
 		log.Fatal(err)
 	}
 
-	for i := 1; i <= 100; i++ {
+	for i := 1; i <= 10; i++ {
```

Comment:
per above I'd leave at 100 for resiliency qs

Reply:
Fixed